notebooks/demo2/teacher_student_exp.ipynb

```diff
@@ -0,0 +1,1862 @@
+{
```

Comment:
<p>We need to update this to reflect real f1 scores</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/248/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>